scipy/stats/_continuous_distns.py

```diff
@@ -6853,7 +6853,12 @@ class trapz_gen(rv_continuous):
     -----
     The trapezoidal distribution can be represented with an up-sloping line
     from ``loc`` to ``(loc + c*scale)``, then constant to ``(loc + d*scale)``
-    and then downsloping from ``(loc + d*scale)`` to ``(loc+scale)``.
+    and then downsloping from ``(loc + d*scale)`` to ``(loc+scale)``.  This
+    defines the trapezoid base from ``loc`` to ``(loc+scale)`` and the flat
+    top from ``c`` to ``d`` proportional to the position along the base
+    with ``0 <= c <= d <= 1``.  When ``c=d``, this is equivalent to `triang`
```

Comment:
Here and in next line, are single backticks supposed to be double?

Reply:
Actually, some of the double backticks should be single.  Following the pattern of other docs, use single backtick to refer to parameters, methods and functions, and double backticks to refer to math expressions.  In this case, I'm referring to `loc`, `scale` and `c` from the `triang` distribution.  Will fix.